pallets/ddc-validator/src/lib.rs

```diff
@@ -911,25 +915,45 @@ pub mod pallet {
 							return Err("signing key not set")
 						}
 						// ToDo: replace local call by a call from `ddc-staking` pallet
-						let _tx_res: Option<(frame_system::offchain::Account<T>, Result<(), ()>)> =
+						let tx_res: Option<(frame_system::offchain::Account<T>, Result<(), ()>)> =
 							signer.send_signed_transaction(|_account| {
 								Call::charge_payments_content_owners {
 									paying_accounts: final_payments.clone(),
 								}
 							});
 
+						let (acc, res) = &tx_res.unwrap();
+
+						match res {
```

Comment:
If `acc` is not needed anywhere else, you may avoid unwrapping by matching on `tx_res` directly.

```rust
match tx_res {
  Some(acc, Ok(())) => {},
  Some(acc, Err(e)) => {},
  _ => (),
}
```